### ISSUE TITLE
resume from a finished checkpoint

### ISSUE THREAD

**Issue by WuXiaoyu55** [milesial/Pytorch-UNet#326]
Hi,

After finishing a run of 30 epochs, I want to resume the training to 200 epochs, How to do that?

Thanks.

**Comment by outsider-duwen** [comment]
> 

Have you solved this problem?

**Comment by kimile599** [comment]
also hope to solve the problem.

**Comment by milesial** [comment]
`train.py` has a `--load` flag. LR scheduler state is not saved, but you can train for 170 epochs from the checkpoint.
